### PR TITLE
GeCraft add OBS chapters/verses

### PR DESCRIPTION
**Pull request by ancientTexts-net**
## Describe what your pull request addresses

- [ ]

## Test Instructions

- [ ]

TODO @ancientTexts-net

### CODE REVIEW

File: src/common/BooksOfTheBible.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-export const NT_ORIG_LANG = 'el-x-koine';
-export const NT_ORIG_LANG_BIBLE = 'ugnt';
-export const OT_ORIG_LANG = 'hbo';
-export const OT_ORIG_LANG_BIBLE = 'uhb';
+export const NT_ORIG_LANG = "el-x-koine";
+export const NT_ORIG_LANG_BIBLE = "ugnt";
+export const OT_ORIG_LANG = "hbo";
+export const OT_ORIG_LANG_BIBLE = "uhb";
```
Comment (PhotoNomad0) [inline src/common/BooksOfTheBible.js:4]: No need to change the string quotes.

**Review by PhotoNomad0 — CHANGES_REQUESTED** [review]
Looks very well done.  Just a some small changes requested.

**Comment by foxprogs** [comment]
@ancientTexts-net . Could you close this pull request? I opened a new one, did it in another branch, otherwise we poured a lot of unnecessary changes from the linter.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "bible-reference-rcl",
-  "version": "0.5.0-rc0",
+  "name": "@texttree/bible-reference-rcl",
```
Comment (PhotoNomad0) [inline package.json:2]: Best not to change the app name in this PR.

**Review by PhotoNomad0 — CHANGES_REQUESTED** [review]

**Comment by PhotoNomad0** [comment]
Closed by request

**Pull request closed without merging**